### PR TITLE
Issue388 - Create language google sheet

### PR DESCRIPTION
**Pull request by alegzandach**

### CODE REVIEW

**Comment by alegzandach** [comment]
This is where the google sheet lives https://docs.google.com/spreadsheets/d/1L633lO5wfQGLbaVnaSnIg9TrprX9k4Ie8qe_CLDmQN8/edit#gid=0

**Comment by Lane** [comment]
@alegzandach this is looking good! would you be able to add the script to fetch the language when running `npm run start` as well?

currently when i pull this branch and run `npm run start` i get an error because `en.js` does not exist:

```
Failed to compile.

./src/modules/explorer/app/selectors/lang.js
Module not found: Can't resolve '../constants/en' in '/Users/lane/Development/seda/map/src/modules/explorer/app/selectors'
```

**Comment by alegzandach** [comment]
I just removed en.js from the gitignore instead of continuously generating a file that im assuming will generally be unchanging. Also re-added _old.

**Review by Lane — CHANGES_REQUESTED** [review]
thanks for that update, one last minor change and it should be good!

File: package.json [hunk 1]
```diff
@@ -41,6 +43,7 @@
     "lodash.debounce": "^4.0.8",
     "lodash.isequal": "^4.5.0",
     "lodash.merge": "^4.6.2",
+    "node-fetch": "^2.6.1",
```
[the remark this answers is not in the document]
Comment (alegzandach) [reply package.json]: got it! done

**Review by Lane — APPROVED** [review]

**Pull request merged**